packages/smart-accounts/src/index.ts

```diff
@@ -0,0 +1,30 @@
+export type * from "./light-account/accounts/account.js";
+export { createLightAccount } from "./light-account/accounts/account.js";
+
+export type * from "./light-account/accounts/multi-owner-account.js";
+export { createMultiOwnerLightAccount } from "./light-account/accounts/multi-owner-account.js";
+
+export type * from "./light-account/decorators/singleOwner.js";
+export { singleOwnerLightAccountActions } from "./light-account/decorators/singleOwner.js";
+
+export type * from "./light-account/decorators/multiOwner.js";
+export { multiOwnerLightAccountActions } from "./light-account/decorators/multiOwner.js";
```

Comment:
Something to consider for v5 - can we cut the exports for the account-specific actions from the public interface, and just consume these internally with account + rely on the standard bundlerClient actions for performing state writes (i.e. rotating owners)? I'm not sure it's useful to consumers outside of attaching to instances of a client with the right action.